po/es.po

```diff
@@ -822,12 +817,10 @@ msgid "embedded flash"
 msgstr "flash incrustado"
 
 #: src/htmlrenderer.cpp:985
-#. An actual tag, no translation needed.
 msgid "video"
-msgstr ""
+msgstr "vídeo"
 
 #: src/htmlrenderer.cpp:987
-#. An actual tag, no translation needed.
 msgid "audio"
 msgstr ""
```

Comment:
Did you forget to add a translation here? You translated "video" above, after all.